Makefile

```diff
@@ -146,6 +157,21 @@ ${1}: export OUT=${OUT}
 ${1}: export OPERATOR_NAME=${2}
 ${1}: export OPERATOR_DIR=${OUT}/${NAMESPACE}/${2}/op
 ${1}: export DEPLOY_DIR=${OUT}/${NAMESPACE}/${2}/cr
+# Openstack Services Containers
+export KEYSTONEAPI_IMG=${KEYSTONEAPI_IMG}
+export MARIADB_DEPL_IMG=${MARIADB_DEPL_IMG}
+export PLACEMENTAPI_IMG=${PLACEMENTAPI_IMG}
+export GLANCEAPI_IMG=${GLANCEAPI_IMG}
+export CINDERAPI_IMG=${CINDERAPI_IMG}
+export CINDERSCHEDULER_IMG=${CINDERSCHEDULER_IMG}
+export CINDERBACKUP_IMG=${CINDERBACKUP_IMG}
+export CINDERVOLUME_IMG=${CINDERVOLUME_IMG}
+export OVNBDS_IMG=${OVNBDS_IMG}
+export OVSBDS_IMG=${OVSBDS_IMG}
+export OVNNORTHD_IMG=${OVNNORTHD_IMG}
+export OVSSERVICE_IMG=${OVSSERVICE_IMG}
+export OVNCONTROLLER_IMG=${OVNCONTROLLER_IMG}
+export NEUTRONSERVER_IMG=${NEUTRONSERVER_IMG}
```

Comment:
Should we perhaps follow the pattern used in the existing vars definitions, i.e. `${1}: export VAR=${VAR}`.  I'm not sure off the top of my head what the functional difference is between that and what we've added here, but I just thought I'd ask.

Reply:
Done